frontend/src/screens/Cart/index.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { ADD_ITEM_TO_CART, REMOVE_FROM_CART } from '../../constants/CartConstants';
+import useCustomContext from '../../customHooks/Hook';
+import styles from './cart.module.css';
+import { Link } from 'react-router-dom';
+import { Message } from '../../components';
+
+const Cart = () => {
```

Comment:
I will suggest changing the file name to Cart.js to be more precise.
And variable names e, m-b-half, p-half, x should be changed to more descriptive names.
You probably have forgotten to remove comments in this file. Please take a look.

Reply:
Done all, but regarding naming file to Cart.js, I think if I renamed it as Cart.js then I need to go 1 level down to import it. And since there is only one file in folder so its not needed I guess!